pages/about.md

```diff
@@ -7,27 +7,3 @@ permalink: /about/
 **Vision:** Our vision is to be the leading inclusive community teaching data and coding skills for HPC resources.
 
 **Mission:** We're still working on our mission statement, but when it lands it's going to be amazing!
-
-#### About the Carpentries in general
-
-The Carpentries project comprises the 
-<a href="https://software-carpentry.org/">Software Carpentry</a>, 
-<a href="http://www.datacarpentry.org/">Data Carpentry</a>, and 
-<a href="https://librarycarpentry.org/">Library Carpentry</a> communities of
-Instructors, Trainers, Maintainers, helpers, and supporters who share a mission
-to teach foundational computational and data science skills to researchers. 
-
-The Carpentries subscribes to a shared
-<a href="http://docs.carpentries.org/topic_folders/policies/code-of-conduct.html">Code of Conduct</a>.
```

Comment:
I would prefer to maintain this section as far as the Code of Conduct, no harm in showing the other Carpentries somewhere

Reply:
Restored by 0ac75b87.